test/esp/integration/json_api_format_integration_test.rb

```diff
@@ -11,12 +11,15 @@ class JsonAPIFormatTest < ESP::Integration::TestCase
 
             assert_not_nil alert.attributes['external_account']
             assert_equal alert.external_account_id, alert.external_account.id
-            assert_not_nil alert.external_account.attributes['organization']
-            assert_equal alert.external_account.organization_id, alert.external_account.organization.id
+
+            assert_nil alert.attributes['team']
             assert_not_nil alert.external_account.attributes['team']
             assert_equal alert.external_account.team_id, alert.external_account.team.id
+
+            assert_nil alert.attributes['organization']
             assert_not_nil alert.external_account.team.attributes['organization']
```

Comment:
That's what the `assert_not_nil ...attributes` assertions are for.  If `alert.attributes['external_account']` is not nil, then calling `alert.external_account` will not make another request.